docs/CHANGELOG.asciidoc

```diff
@@ -93,6 +94,279 @@ you make the necessary updates after you upgrade to 8.15.0.
+Observability::
+* Updates links for integration buttons in Observability solution ({kibana-pull}184477[#184477]).
+* Adds SLO status, SLI value, error budget remaining and consumed to the burn rate alert context ({kibana-pull}184471[#184471]).
+* Adds an option to prevent initial backfill for SLOs ({kibana-pull}184312[#184312]).
+* Updates `Add data` links to use improved deep linking ({kibana-pull}184164[#184164]).
+* Changes the navigation behavior for the Observability guide cards to pre-select the correct solution ({kibana-pull}184065[#184065]).
+* Updates the Alerts details to now show an history chart for all types of alerts ({kibana-pull}181824[#181824]).
+* Adds Docs count, Size, Services, Hosts, and Degraded docs KPIs to the dataset quality flyout ({kibana-pull}179479[#179479]).
+Platform::
+* Improves the **Share** menu to let you navigate through a tabbed modal to copy links for Discover, Dashboards, and Lens. ({kibana-pull}180406[#180406]).
+* Changes the behavior of the "Endpoints and API keys" button in the header to open the Connection details flyout ({kibana-pull}183236[#183236]).
+* Adds a quick way to create an API keys with a 90-days expiration to the Connection details flyout, and clarifies the Elasticsearch endpoint and Cloud ID information ({kibana-pull}180912[#180912]).
+* Adds a feedback button to the header in Serverless projects({kibana-pull}180942[#180942]).
+
+For more information about the features introduced in 8.15.0, refer to <<whats-new,What's new in 8.15>>.
+
+[[enhancements-and-bug-fixes-v8.15.0]]
+=== Enhancements and bug fixes
+
+For detailed information about the 8.15.0 release, review the enhancements and bug fixes.
+      
+
+[float]
+[[enhancement-v8.15.0]]
+=== Enhancements
+Alerting::
+* Adds support of additional fields for ServiceNow ITSM and SecOps ({kibana-pull}184023[#184023]).
+* Adds support for the additional info field in the ServiceNow ITOM connector ({kibana-pull}183380[#183380]).
+Cases::
+* The Cases webhook connector now supports SSL certificate authentication ({kibana-pull}185925[#185925]).
+Dashboards::
+* Adds a "Creator" column to the Dashboards list ({kibana-pull}182256[#182256]).
+* Adds the ability to filter dashboards by creator for dashboards created on or after version 8.14 ({kibana-pull}180147[#180147]).
+* When switching back from maximized to minimized view on a panel, you return to your original position in the dashboard ({kibana-pull}184696[#184696]).
+* Improves the dashboard background color to match the current color mode wen margins are turned off ({kibana-pull}181450[#181450]).
+* Simplifies the workflow for creating a copy of the dashboard currently open in both view and edit modes ({kibana-pull}180938[#180938]).
+Data Discovery::
+* CSV reports now include custom field labels when they exist ({kibana-pull}181565[#181565]).
+Discover::
+* Adds the ability to creates ES|QL WHERE filters for ordinal charts ({kibana-pull}184420[#184420]).
```

Comment:
Can we move it to the section above (features) and merge it with 
```
In ES|QL mode, you can now create WHERE clause filters more intuitively by interacting with the table, sidebar and table row viewer
```
or if not merging them, def group them together